packages/front-end/styles/global.scss

```diff
@@ -128,6 +128,13 @@ pre {
   }
 }
 
+#metric-page-tabs {
+  font-size: 1.2em;
+  .active-tab {
+    color: var(--text-color-primary);
+  }
+}
```

Comment:
I would remove this.  We're going to switch tabs to use Radix soon, and any custom styling just makes that migration more difficult.